ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```diff
@@ -86,6 +126,33 @@
     - adoptopenjdk
   when: not solaris_studio.stat.exists
 
+- name: Check if freetype is installed (sparcv9)
```

Comment:
Can you add a comment about why this is only for SPARC? Seems odd that it wouldn't be in pkguntil the repositories if it is for x64, so we should make it clear what the reason is.

Reply:
done